.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,64 @@
+---   # Proof generation benchmarking workflow
+
+name: Benchmark proving
+
+on:
+  schedule:
+    - cron: "0 4 * * *"
+  workflow_dispatch:
+    branches:
+      - "**"
+
+env:
+  BENCHMARK_WITNESS: artifacts/witness_b1000_b1019.json
+
+jobs:
+  benchmark_proving:
+    name: Benchmark proving for representative blocks
+    runs-on: zero-reg
```

Comment:
info: is there somewhere I can read about the different CI runners we have, and how they're spec'd out/intended to be used?

Reply:
This one is created by Ben for this particular benchmark